sdk/appplatform/Azure.ResourceManager.AppPlatform/api/Azure.ResourceManager.AppPlatform.netstandard2.0.cs

```diff
@@ -0,0 +1,2133 @@
+    }
+    public partial class SupportedStackResourceCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.AppPlatform.SupportedStackResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.AppPlatform.SupportedStackResource>, System.Collections.IEnumerable
+    {
+        protected SupportedStackResourceCollection() { }
+        public virtual Azure.Response<bool> Exists(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.AppPlatform.SupportedStackResource> Get(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.AppPlatform.SupportedStackResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.AppPlatform.SupportedStackResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.AppPlatform.SupportedStackResource>> GetAsync(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.AppPlatform.SupportedStackResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.AppPlatform.SupportedStackResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.AppPlatform.SupportedStackResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.AppPlatform.SupportedStackResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class SupportedStackResourceData : Azure.ResourceManager.Models.ResourceData
+    {
+        public SupportedStackResourceData() { }
+        public Azure.ResourceManager.AppPlatform.Models.SupportedStackResourceProperties Properties { get { throw null; } set { } }
+    }
+}
+namespace Azure.ResourceManager.AppPlatform.Models
+{
+    public partial class ActiveDeploymentCollection
```

Comment:
Avoid using `Collection` as model suffix.

Reply:
too many for manual fixing, so would like the codegen can auto-resolve these kind model name, here is an issue opened for this scenario https://github.com/Azure/autorest.csharp/issues/2279